src/patients/list/ViewPatients.tsx

```diff
@@ -91,6 +102,34 @@ const ViewPatients = () => {
     setSearchText(event.target.value)
   }
 
+  if (patients && patients.length === 0) {
+    return (
+      <div style={{ display: 'flex', justifyContent: 'center' }}>
+        <div style={{ display: 'flex', flexDirection: 'column' }}>
+          <div style={{ display: 'flex', justifyContent: 'center' }}>
+            <Icon icon="patient" outline={false} size="6x" />
+          </div>
+          <div style={{ display: 'flex', justifyContent: 'center' }}>
+            <div style={{ textAlign: 'center', width: '60%', margin: 16 }}>
+              <Typography variant="h5">{t('patients.noPatients')}</Typography>
+            </div>
+          </div>
+          <div style={{ display: 'flex', justifyContent: 'center' }}>
+            <Button
+              key="newPatientButton"
+              outlined
+              color="success"
+              icon="patient-add"
+              onClick={() => history.push('/patients/new')}
+            >
+              {t('patients.newPatient')}
+            </Button>
+          </div>
+        </div>
+      </div>
```

Comment:
I think that we should move this to its own component to: 
- keep components smaller and easier to understand
- make it easier to test